challenges/Front-End Challenge/src/components/Form.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import { ReCaptcha } from 'react-recaptcha-v3'
+
+const FormDiv = styled.form`
+    margin: 10% 20%;
+    width: 60%;
+    display: inline-block;
+    text-align: center;
+    margin-bottom: 40vh;
+`;
+
+
+const SubmitButton = styled.div`
+    cursor: pointer;
+    text-align: center;
+    vertical-align: middle;
+    border: 1px solid rgb(216, 216, 216);
+    border-radius: 0.5em;
+    font-size: 120%;
+    width: 10%;
+    padding: 1%;
+    min-width: 110px;
+    min-height: 50px;
+    padding-top: 20%;
+
+    transition: transform 750ms;
+
+    &:hover {
+    transform: scale(1.25);
+    }
+`;
+
+const MessageResponse = styled.div`
+    text-align: center;
+    margin-top: 1%;
+`;
+
+
+export default function Form() {
+    const siteKey = '6Le3WMAUAAAAAOCR9AMk7lEj-N8S_bo-nLfRKOXW';
+    const secretKey = '6Le3WMAUAAAAABGIP81B-wOoxHU3iUTTshD7Zlw8';
```

Comment:
I would like to point out that this is not standard with accessing API keys. Normally you would make a secure request to a protected server to retrieve this key without giving away the value.